workstation.yml

```diff
@@ -60,6 +60,7 @@
   vars:
     os_manila_mount_shares:
       - share_name: azimuth-project-share
+        share_user: "{{ cluster_project_manila_share_user }}"
```

Comment:
I think we need to default these two to None or similar? Because when we only pass them when cluster_project_manila_share = true.